### DISCUSSION TITLE
Error in installing the  smartcontractkit/chainlink-brownie-contracts

### DISCUSSION THREAD

**Discussion by d4r3-w0lf** [Cyfrin/foundry-full-course-cu#2288] [General]
When running the command:

forge install smartcontractkit/chainlink-brownie-contracts@0.6.1 --no-commit

It is giving an error:

remote: Enumerating objects: 34274, done.
remote: Counting objects: 100% (3875/3875), done.
remote: Compressing objects: 100% (1140/1140), done.
error: 1855 bytes of body are still expectediB | 37.00 KiB/s 
fetch-pack: unexpected disconnect while reading sideband packet
fatal: early EOF
fatal: fetch-pack: invalid index-pack output

Can anyone help me?

**Comment by EngrPips** [comment]
Hello @Nandeesh-D, This is an internet connection problem. Please try again when your internet connection is more stable.

**Comment by d4r3-w0lf** [comment]
Thank you I'll try it

**Reply by EngrPips** [reply]
You are welcome.

**Comment by KevinFalah** [comment]
Yeaa i got same issue when i use my wifi and then i change with my mobile hotspot and its worked

**Reply by EngrPips** [reply]
Interesting.
